python/graphstorm/config/argument.py

```diff
@@ -1249,9 +1249,9 @@ def num_classes(self):
             "Must provide the number possible labels through num_classes"
         if isinstance(self._num_classes, dict):
             for num_classes in self._num_classes.values():
-                assert num_classes > 1
+                assert num_classes > 0
```

Comment:
Can you write some comments here to explain why?

Reply:
I added a comments in the code. Please check.